navigation/pipelines_dashboard_page.md

```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
 
 ### Key
 
-1.  The pipelines are listed under the pipeline group that they belong to
-2.  Click the pipeline name to see [pipeline activity](../navigation/pipeline_activity_page.md) for that pipeline.
-3.  Click pipeline instance label to see details for that instance.
-4.  Click on each stage segment to see [stage details](../navigation/stage_details_page.md) for that stage instance.
-5.  "triggered by [user name] about [how long ago]" gives you a quick look at who activated this pipeline and when this was triggered.
-6.  The "Trigger" button forces a pipeline to begin build activity
-7.  The [Trigger with Options](../advanced_usage/trigger_with_options.md) button allows the you to pick the revisions of materials that the pipelines should build with, and trigger the pipeline.
-8.  The "Pause" button pauses scheduling of the pipeline.
+1.  The pipelines are listed under the pipeline group that they belong to. You can also edit pipeline group by clicking on settings icon.
+2.  Pipeline instance [label](../configuration/build_labelling.md) which can be equivalent to pipeline run. 
+3.  Click on each stage segment to see [stage details](../navigation/stage_details_page.md) for that stage instance.
+4.  The "Trigger" button forces a pipeline to begin build activity.
+5.  The [Trigger with Options](../advanced_usage/trigger_with_options.md) button allows the you to pick the revisions of materials that the pipelines should build with, and trigger the pipeline.
+6.  The "Pause" button pauses scheduling of the pipeline.
+7.  If you are a GoCD pipeline group administrator or a super administrator, you can now navigate to edit a pipeline by clicking this settings icon on the pipeline dashboard or the environments page.
```

Comment:
I'd remove the part about the environments page. That pipeline view is actually scheduled to go away soon.